apps/expo/src/components/notif/dmNotifs/dmNotifs.tsx

```diff
@@ -1,12 +1,14 @@
 import { View, Text } from "react-native"
 import { useGlobalContext } from "~/context/global-context";
 import { api } from "~/utils/api"
-import Conversation from "./Conversation";
-import RotatingBarbellIcon from "./RotatingBarbellIcon";
+import Conversation from "apps/expo/src/components/notif/Conversation";
```

Comment:
Can you change this to imports with ~ because it will break the testing with jest as it uses the ~ for imports.